readme.md

```diff
@@ -43,10 +43,10 @@ del.sync(['public/assets/**', '!public/assets', '!public/assets/goat.png']);
 
 ## API
 
-### del(patterns, [options], callback)
+### del(patterns, [options])
 ### del.sync(patterns, [options])
 
-The async method gets an array of deleted paths as the second argument in the callback, while the sync method returns the array.
+The async method gets Promise yielding an array of deleted paths, while the sync method returns the array.
```

Comment:
For consistency with the globby readme, can you use `resolves` instead of `yielding`? (Or is `yielding` more correct?).